parts/linux/cloud-init/artifacts/cse_config.sh

```diff
@@ -306,6 +306,11 @@ ensureContainerd() {
   if [ "${TELEPORT_ENABLED}" == "true" ]; then
     ensureTeleportd
   fi
+  if [ "${ARTIFACT_STREAMING_ENABLED}" == "true" ]; then
+    echo "ARTIFACT STREAMING is enabled, running test function for artifact streaming"
+    ensureArtifactStreaming
```

Comment:
```suggestion
    logs_to_events ensureArtifactStreaming
```

maybe we want to log this as an actual CSE step?